build_tools/cmake/iree_python.cmake

```diff
@@ -234,19 +234,25 @@ function(iree_local_py_test)
       ${_RULE_ARGS}
   )
 
-  list(APPEND _RULE_PACKAGE_DIRS "$ENV{PYTHONPATH}")
-  string(JOIN ":" _PYTHONPATH ${_RULE_PACKAGE_DIRS})
-
-  if (NOT DEFINED _RULE_TIMEOUT)
-    set(_RULE_TIMEOUT 60)
-  endif()
-
   set_property(TEST ${_NAME_PATH} PROPERTY LABELS "${_RULE_LABELS}")
   set_property(TEST ${_NAME_PATH} PROPERTY TIMEOUT ${_RULE_ARGS})
 
+  # Extend the PYTHONPATH environment variable with _RULE_PACKAGE_DIRS.
+  list(APPEND _RULE_PACKAGE_DIRS "$ENV{PYTHONPATH}")
+  if(${CMAKE_SYSTEM_NAME} STREQUAL "Windows")
+    # Windows uses semi-colon delimiters, but so does CMake, so escape them.
```

Comment:
:cry: Luckily this doesn't get passed around too much afterwards. Is it really necessary though? I thought when you expanded a list within quotes you got the string version anyway. I'm guessing you determined that it is necessary, but I'm slightly surprised

Reply:
Yeah, I tried a few variations on this and either got only the first element of the list (i.e. `runtime`) or no delimiter at all (i.e. `runtimecompiler`).